src/Physics/physicsHelper.ts

```diff
@@ -411,20 +436,19 @@ export class PhysicsUpdraftEvent {
 
     /**
      * Initializes the physics updraft event
+     * @param _physicsHelper A physics helper
      * @param _scene BabylonJS scene
      * @param _origin The origin position of the updraft
-     * @param _radius The radius of the updraft
-     * @param _strength The strength of the updraft
-     * @param _height The height of the updraft
-     * @param _updraftMode The mode of the updraft
+     * @param _options The options for the updraft event
      */
-    constructor(private _scene: Scene, private _origin: Vector3, private _radius: number, private _strength: number, private _height: number, private _updraftMode: PhysicsUpdraftMode) {
+    constructor(private _physicsHelper: PhysicsHelper, private _scene: Scene, private _origin: Vector3, private _options: PhysicsUpdraftEventOptions) {

```

Comment:
This is also a breaking change (perhaps just moving the helper at the end then

Reply:
That was rather a consistency fix, because only the `PhysicsGravitationalFieldEvent` had the ˙PhysicsHelper` in the constructor, while other events didn't have it. Either way, it would be a breaking change, if we want consistency with the signatures.
Now, the next thing here is, that you are not really supposed to ever call that by your own, but rather via the calls (`applyRadialExplosionImpulse`, ...), as it does some of the actual processing happens there.
So I guess the best option would be to just not export/expose these classes publicly, as they aren't supposed to be public? Sorry, didn't think of that before.

Reply:
no worries :) I'm fine to makr then private then

Reply:
Ok, that's fixed now!